.gitignore

```diff
@@ -29,5 +29,8 @@ node_modules
 # Jetbrains IDEs
 .idea
 
+# vim swp files
```

Comment:
just like .idea above, this also seems like something that can't hurt, but should be in your global gitignore if you use vim, rather than spilling out into every project you contribute to :-/